css/giving-rides.css

```diff
@@ -0,0 +1,304 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul[class],
+ol[class],
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
+  margin: 0;
+}
+
+html{
+    /* font-size:62.5%; */
+    text-decoration: none;
+    font-family: 'Open Sans', sans-serif;
+}
+ul{
+    list-style-type: none;
+    margin: 0;
+}
+
+.important-info{
+    width: auto;
+    height: 8.5rem;
+    background-color: #DFF4F3;
+    font-size: 2.6rem;
+    display: flex;
+    align-items: center;
+    padding-left:11rem; 
+}
+#info-list li::before{
+    content: "\25A0";
+    color: #5C6E93;
+    display: inline-block; /* Needed to add space between the bullet and the text */
+    width: 5rem; /* Also needed for space (tweak if needed) */
+    padding-top: 2rem;
+    margin-left: -5rem; /* Also needed for space (tweak if needed) */
+}
+#info-list{
+    margin-top: 2rem;
+    display:flex;
+    /* list-style-type: square; */
+    flex-direction:column;
+    font-size: 1.6rem;
+    margin-left: 37rem;
```

Comment:
Instead of pushing everything to the right, you can center it using margin: 0 auto. However, when you use this property you need to make sure the width isn't the entire screen width or else it won't be able to center anything (if the width is the whole screen width it will default be "centered" already). You can use width: fit-content; so the width becomes whatever the content is and not the whole screen. You also might run into the issue where you're trying to center the UL, but it's only centering the text and not including the boxes. Try googling a sol'n to that and LMK if you have Qs on any of this / if this is unclear